scripts/lib/sphinx/sphinxHtmlToMarkdown.ts

```diff
@@ -581,3 +507,41 @@ function buildSpanId(id: string): MdxJsxFlowElement {
     children: [],
   };
 }
+
+// translate type headings to titles
```

Comment:
This comment is confusing because the function is more generic than this. I recommend using proper docstring and something like this:

> Find the element that both matches the `selector` and whose content is the same as `text`.

No need to mention how this function gets used. It could be used for many purposes.

--


Also the return type should be `Cheerio`. Ditto that all the functions should have a return type.

Reply:
I agree. I kept all the original comments like that one, and it's confusing. I was planning to add the docstring to all functions and improve all the comments in a follow-up. This one, in particular, was inside the "main" function before, and it's only copied outside. Nonetheless, I added the comment you wrote, thanks :+1: